sdk/mixedreality/mixedreality-authentication/src/util/base64.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { isNode } from "@azure/core-http";
+
+declare global {
+  // stub these out for the browser
+  function btoa(input: string): string;
+  function atob(input: string): string;
+}
+
+/**
+ * Base64 encode.
+ *
+ * @export
+ * @param {string} content
+ * @returns {string}
+ */
+export function base64encode(content: string): string {
```

Comment:
@xirzec 
This looks like another reason to have `core-encoding` :smile:

Reply:
It is tempting. Are there other encodings we've been needing to support?

Reply:
Yes please!